docs/sources/static/_index.md

```diff
@@ -19,7 +19,7 @@ Static mode is composed of different _subsystems_:
   traces and forwarding them to Grafana Tempo or any OpenTelemetry-compatible
   endpoint.
 
-Static mode is [configured][] with a YAML file.
+Static mode is [configured][configure] with a YAML file.
```

Comment:
How does this work? What is shown to the user?

Reply:
> How does this work? What is shown to the user?

It's in the format `[link text][label]`. The link text is what's shown to the reader, and the label is what Hugo uses to connect the link text with a URL. So, you can use different link text throughout the topic if you reuse the link destination (defined by the label).

In this specific case, the reader sees `Static mode is configured with a YAML file.` and the link under `configured` is defined on line 89/90 of this file as `[configure]: "/docs/agent/ -> /docs/agent/<AGENT VERSION>/static/configuration"`

https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference-shortcode